modules/shared_vpc_access/variables.tf

```
@@ -58,3 +58,9 @@ variable "grant_services_security_admin_role" {
   type        = bool
   default     = false
 }
+
+variable "grant_services_network_role" {
+  description = "Whether or not to grant the service accounts the network roles on the host project so they can manage firewall rules"
```

nit: since we are gating `networkUser` role and service agent related roles which do not manage FW rules
```suggestion
  description = "Whether or not to grant service agents the network roles on the host project"
```